runhouse/main.py

```diff
@@ -679,6 +350,182 @@ def cluster_list(
         print_sky_clusters_msg(len(sky_clusters))
 
 
+@cluster_app.command("keep-warm")
+def cluster_keep_warm(
+    cluster_name: str,
+    mins: Optional[int] = typer.Option(
+        -1, help="Amount of time (in min) to keep the cluster warm after inactivity."
+    ),
+):
+    """Keep the cluster warm for given number of minutes after inactivity.
+
+    Example:
+        ``$ runhouse cluster keep-warm /sashab/rh-basic-cpu``
+
+    """
+
+    try:
+        current_cluster = rh.cluster(name=cluster_name)
+        if not current_cluster.is_up():
+            print_bring_cluster_up_msg(cluster_name=cluster_name)
+            return
+        current_cluster.keep_warm(mins=mins)
+        # TODO: keep warm when we are in the cluster?
+    except ValueError as e:
+        if str(e) == f"Resource {cluster_name} not found":
+            console.print("Cluster is not saved in den.")
+            # TODO: do we want to check if the cluster is a live sky cluster, and inform the user how to keep_warm using sky?
+        else:
+            console.print(f"Failed to keep the cluster warm: {e}")
+    except Exception as e:
+        console.print(f"Failed to keep the cluster warm: {e}")
+
+
+# TODO [SB/JL]: uncomment and update once Rohin's change is merged.
```

Comment:
dumb nit but can we move this further down just for readability purposes

Reply:
also i think we can just remove all the logic here and just have "pass" instead for now